backend/src/database/migrations/20230408151021-add-api-key-expiry-and-contact.js

```diff
@@ -0,0 +1,21 @@
+'use strict'
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.addColumn('api_keys', 'valid_until', {
+      type: Sequelize.DataTypes.DATE,
+      allowNull: false,
+      defaultValue: new Date('2024-04-21T00:00:00.000Z'),
```

Comment:
Do we want to use a default time that is more intuitive to users in Singapore?

Reply:
E.g. 2024-04-20 11:59pm GMT+8

Reply:
actually, now that I look at it `21 April 2024, 8am` seems pretty intuitive to me too

<img width="108" alt="image" src="https://user-images.githubusercontent.com/67887489/232382645-247809c9-d967-45b6-a7bc-1adec94f8c0e.png">